accounts-db/src/tiered_storage/hot.rs

```diff
@@ -227,6 +227,17 @@ impl HotStorageReader {
         let (meta, _) = get_type::<HotAccountMeta>(&self.mmap, offset)?;
         Ok(meta)
     }
+
+    /// Returns the offset of the account associated with the specified index.
+    fn get_account_offset(&self, index: usize) -> usize {
```

Comment:
Updated the comment.  The offset to the account in the underlying TieredStorageFile.

As for creating a new type, does `TieredStorageFileOffset` or simply `TieredStorageOffset` look good?

Reply:
It is a raw byte-offset from the beginning of the file (since the accounts block is located at the beginning of a tiered-storage file.)

As for a new type, I have created a https://github.com/solana-labs/solana/pull/33872 that defines `TieredStorageOffset`.